helm/values.yaml

```diff
@@ -144,12 +144,12 @@ cronJob:
     size: 5Gi
   resources:
     requests:
-      cpu: "50m"
-      memory: "256Mi"
+      cpu: '50m'
```

Comment:
just curious what is converting the single to double quotes. as i'm sure we don't have a script that lints these files, and it is an influence of VS Code autosaving this.  In either case we should be cautious.

Reply:
https://github.com/bcgov/des-notifybc/blob/v5/.prettierrc.yml#L1

I'm not sure why they weren't single quotes in the first place.